packages/dhis2/src/Adaptor.js

```diff
@@ -279,8 +304,8 @@ export function create(resourceType, data, options = {}, callback = false) {
  *   storedBy: 'admin',
  *   dataValues: [],
  * });
- * @example <caption>a trackedEntityInstance</caption>
- * update('trackedEntityInstances', 'IeQfgUtGPq2', {
+ * @example <caption>a trackedEntityInstance. Update a tracker instance. See {@link https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-241/tracker.html#webapi_nti_import Update tracker docs}</caption>
```

Comment:
This caption is wierd.

We're not going to re-write all the captions in this  work. I suggest we simplify and just leave the example the same (but say `a tracked entity` rather than `a trackedEntityInstance`)